Update new project section (#57)
django-commons/controls: README.md

           is_django_commons_repo = optional(bool, false) # Do not create teams for repository
           enable_branch_protection = true # optional, default is true
           required_status_checks_contexts = [] # optional, default is []
           admins = [] # Members of the repository's admin and repository teams. Have admin permissions
           committers = [] # Members of the repository's committers and repository teams. Have write permissions
           members = [] # Members of the repository team. Have triage permissions
         }
       }
       ```

    - [ ] Create a pull-request to `main` branch. This will trigger terraform to plan the changes in the organization to
      be executed.
      Review the changes and make sure they align with the request.
    - [ ] Merge the pull request. This will trigger terraform to apply the changes in the organization.
    - The expected changes:
        - [ ] New teams `repo-name`, `repo-name-admins`, `repo-name-committers` with the relevant members based on the
          repository's description.
        - [ ] The repository changes are accepted by the project maintainers.

>         - [ ] Repository has two environments: `pypi` and `testpypi`, see example [here][playground-enviroments]

Reviewer: ```suggestion
        - [ ] Repository has two environments: `pypi` and `testpypi`, see example [here][playground-environments]
```

- [ ] Repo changes:
    - [ ] (project owner) Create/Update the release GitHub workflow in the repository, example can be
      found [here][release-gh-workflow]
    - [ ] Under Actions > General > "Fork pull request workflows from outside collaborators", set "Require approval for
      first-time contributors"

> - [ ] pypi and test-pypi changes:

Reviewer: ```suggestion
- [ ] PyPI and Test PyPI changes:
```

>     - [ ] Add the release workflow to pypi's package publishing (and test-pypi's package publishing).

Reviewer: ```suggestion
    - [ ] Add the release workflow to pypi.org's package publishing (and test.pypi.org's package publishing).
```

## New Project Playbook

Assuming the repository name is `repo-name`:

### Pre Transfer Steps

- [ ] Check if the repository meets [inbound requirements][3].
- [ ] Confirm who will be the admins and maintainers for the repository
- [ ] Make sure the there are no teams `{repo-name}`, `{repo-name}-admins` and `{repo-name}-committers` in the Django
  Commons organization. Teams can be viewed [here][teams]. The teams will be created by the terraform apply process.

> - [ ] (project owner) PyPI project owner must add the Django Commons pypi's team members as owners in [PyPI][pypi],

Reviewer: ```suggestion
- [ ] (project owner) PyPI project owner must add the Django Commons PyPI Admins as owners in [PyPI][pypi],
```

  and [test-pypi][test-pypi]

>   (TODO: Determine how this works with transferring out of an org and into the Django Commons org)

Reviewer: ```suggestion
```

- The repository's teams will be removed from the organization.

[1]: https://github.com/django-commons/membership/blob/main/terraform/production/org.tfvars

[2]: https://github.com/django-commons/membership/blob/main/terraform/production/repositories.tfvars

[3]: https://github.com/django-commons/membership/blob/main/incoming_repo_requirements.md

[people]: https://github.com/orgs/django-commons/people

[teams]: https://github.com/orgs/django-commons/teams

[test-pypi]: https://test.pypi.org/manage/project/django-commons/

[pypi]: https://pypi.org/

[gh-docs-transfer-repo]: https://docs.github.com/en/repositories/creating-and-managing-repositories/transferring-a-repository

[release-gh-workflow]: https://github.com/django-commons/django-commons-playground/blob/main/.github/workflows/release.yml

[pypi-publishing]: https://test.pypi.org/manage/project/django-tasks-scheduler/settings/publishing/

> [playground-enviroments]: https://github.com/django-commons/django-commons-playground/settings/environments

Reviewer: ```suggestion
[playground-environments]: https://github.com/django-commons/django-commons-playground/settings/environments
```
